go/cmd/vtctldclient/internal/command/schema.go

```diff
@@ -29,12 +29,24 @@ import (
 	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
 )
 
-// GetSchema makes a GetSchema gRPC call to a vtctld.
-var GetSchema = &cobra.Command{
-	Use:  "GetSchema [--tables TABLES ...] [--exclude-tables EXCLUDE_TABLES ...] [{--table-names-only | --table-sizes-only}] [--include-views] alias",
-	Args: cobra.ExactArgs(1),
-	RunE: commandGetSchema,
-}
+var (
+	// GetSchema makes a GetSchema gRPC call to a vtctld.
+	GetSchema = &cobra.Command{
+		Use:  "GetSchema [--tables TABLES ...] [--exclude-tables EXCLUDE_TABLES ...] [{--table-names-only | --table-sizes-only}] [--include-views] alias",
+		Args: cobra.ExactArgs(1),
+		RunE: commandGetSchema,
+	}
+	// ValidateSchemaKeyspace makes a ValidateSchemaKeyspace gRPC call to a vtctld.
+	ValidateSchemaKeyspace = &cobra.Command{
+		Use:  "ValidateSchemaKeyspace",
```

Comment:
`<keyspace>` should be a positional arg, imo. So the usage should look like: `ValidateSchemaKeyspace [flags] <keyspace>`